pkg/document/editor/editor.go

```diff
@@ -17,17 +18,19 @@ const (
 	DocumentID     = "id"
 )
 
-func Deserialize(data []byte, identityResolver *identity.IdentityResolver) (*Notebook, error) {
+func Deserialize(log *zap.Logger, data []byte, identityResolver *identity.IdentityResolver) (*Notebook, error) {
```

Comment:
Suggestion: as this interface will likely get extended, I suggest this:

```suggestion
type Options struct {
    IdentityResolver *identity.IdentityResolver
    Logger *zap.Logger
}

func Deserialize(data []byte, opts Options) (*Notebook, error) {
```